example/autowire_redis_client/cmd/main.go

```diff
@@ -93,9 +102,28 @@ func (a *App) Run() {
 		panic(err)
 	}
 	fmt.Println("client3 get ", val4)
+
+	val5, err := a.NormalDB4Redis.Get("mykey").Result()
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println("client4 get ", val5)
+
+	val6, err := a.NormalDB5Redis.Get("mykey").Result()
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println("client5 get ", val6)
+}
+
+func init() {
+	_ = os.Setenv("REDIS_ADDRESS_EXPAND", "localhost:6379")
 }
 
 func main() {
+	// wd, _ := os.Getwd()
+	// wd = filepath.Join(wd, "./autowire_redis_client/cmd")
+	// wd = filepath.Join(wd, "../conf")
```

Comment:
Can we remove these comments or describe them clearly.?

Reply:
I tested on both win and mac: `../conf` would point to a wrong path(e.g.: $GOPATH/src/github.com/alibaba/IOC-Golang/conf/ioc_golang.yaml). 

So there is a write of `wd, _ := os.Getwd()`.